tests/test_leap_seconds.py

```
@@ -25,226 +25,215 @@
-        insert_leap_second(2020,9,3)
-        self.assertEqual(leapsecs_from_ym(2021,1), count + 4)
-
-        insert_leap_second(2020,11,-4)
-        self.assertEqual(leapsecs_from_ym(2021,1), count)
-
-        # Restore without added leap seconds
-        load_lsk()
-        self.assertEqual(leapsecs_from_ym(2021,1), count)
-        self.assertEqual(leap_seconds._SELECTED_UT_MODEL, 'LEAPS')
-        self.assertIs(leap_seconds._SELECTED_DELTA_T, leap_seconds._DELTA_T_DICT['LEAPS'])
+def test_leap_seconds():
```

Ruff (0.15.6)

[warning] Too many statements (115 > 50)

(PLR0915)